.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Request creation of a new format specification'
+labels: spec-request
+---
+<!--
+A request for a format specification consists of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete fields you cannot populate and all the comments from this template (including this one!).
+The resulting YAML blocks should contain valid YAML.
+Mandatory fields are marked with the word MANDATORY for your convenience.
+-->
+```yaml
+# This block is the boilerplate of a KSY spec file. It must be a valid KSY. It is meant to be ready to be copied as it is in order to make the task of the implementer easier.
+meta: # see https://doc.kaitai.io/ksy_reference.html#meta for more info
+  id: # insert an ID you feel right to assign to the format here. KS currently has no namespaces, so the id MUST be unique within all the repo and you should be sure there will be no collisions. If in doubt, add a prefix of an app name. MANDATORY.
+  title: # insert a human-readable name of the format here. MANDATORY.
+  application: # insert a human-readable name of the application for which this format is native here. If it is native for several applications, choose the ones you feel the most affiliated with the format. If the format is a widespread one, like PNG or WAV, just remove this field.
+  file-extension:
+    # insert here a YAML list of file extensions should be placed here.
+  xref: # check if the following sources contain info about the format you request. See https://doc.kaitai.io/user_guide.html#meta for more info
+    mime: # MIME type
+    rfc: # Link to RFC
+    din:
+      id: # DIN standard identifier
+    iso: # ISO standard identifier
+    forensicswiki: # Page name on https://forensicswiki.org/
+    justsolve: # ID on http://fileformats.archiveteam.org/wiki/
+    pronom: # IDs on https://www.nationalarchives.gov.uk/pronom/fmt/
+     - x-fmt/ 
+    wikidata: Q # ID on WikiData
+    fileformat: # Link to https://www.fileformat.info
+    loc: # ID in https://www.loc.gov/preservation/digital/formats/    din:
+      id: # DIN standard identifier
+
+doc: |
+  # Describe your format briefly AND add links to the sources of sample files in the Web. MANDATORY
+
+doc-ref:
+  # A YAML array of URIs to information sources and/or their names goes here. MANDATORY. See https://doc.kaitai.io/ksy_reference.html#doc-ref for more info on sources which are not just an URI
```

Comment:
I would assume that a regular outside person has no idea what a YAML array is, and, at this point, they probably don't care.

Reply:
KS is currntly YAML based (though it could be anything-JSON-compatible-based). Without some basic knowledge of YAML they cannot contribute and even discuss meaningfully the specs. So IMHO some basic knowledge of YAML is prerequisite for working with the project itself. And YAML is not very complex and quite similar to JSON, most of target audience knows JSON. Should we find a good YAML cheatsheet and add a link?